lib/graphql_practice_web/resolvers/user.ex

```diff
@@ -16,12 +17,32 @@ defmodule GraphqlPracticeWeb.Resolvers.User do
   def create_user(_, params, _) do
     case Accounts.create_user(params) do
       {:error, changeset} ->
-        {:error,
-         message: "Could not create user!",
-         details: Ecto.Changeset.traverse_errors(changeset, fn {msg, _opts} -> msg end)}
+        {:error, changeset}
 
       {:ok, user} ->
         {:ok, %{user: user}}
```

Comment:
Could put these back to single lines if you want.

Reply:
I guess :)